doc/PROTOCOL-HTTP2.md

```diff
@@ -61,6 +61,8 @@ the form shown above.
 
 If **Timeout** is omitted a server should assume an infinite timeout. Client implementations are free to send a default minimum timeout based on their deployment requirements.
 
+If **Content-Type** does not begin with "application/grpc", gRPC servers SHOULD respond with HTTP status of 400 (Bad Request).  This will prevent other HTTP/2 clients from interpreting a gRPC error response, which uses status 200 (OK), as successful.
```

Comment:
Should we also modify the ABNF rule on line 110 since it seems to imply only 200 OK is returned?

Reply:
The way I think of it is: the ABNF rules above cover the gRPC protocol, whereas this case is about sending a non-gRPC response to a non-gRPC request.  Changing the ABNF rules will unnecessarily confuse the protocol definition, which always expects status 200 when things are configured correctly.

If the majority feel otherwise, I can change it.

Reply:
Hmmm, okay I can see that, although this is subjective as you'll agree.